worldengine/cli/main.py

```diff
@@ -356,13 +363,18 @@ def main():
     # -----------------------------------------------------
     export_options = parser.add_argument_group(
         "Export Options", "You can specify the formats you wish the generated output to be in. ")
-    export_options.add_argument("--export-format", dest="export_format", type=str,
-                                help="Export to a specific format such as BMP or PNG. " +
-                                "See http://www.gdal.org/formats_list.html for possible formats.",
-                                default="PNG")
-    export_options.add_argument("--export-datatype", dest="export_datatype", type=str,
-                                help="Type of stored data, e.g. uint16, int32, float32 etc.",
-                                default="uint16")
+    export_options.add_argument("--export-type", dest="export_type",
```

Comment:
These are undone changes from a PR of mine that was pulled yesterday, I believe. Did you maybe do a rebase and merge in the wrong set of changes, i.e. the old code?